### PR TITLE
lt-Speech-components

### PR DESCRIPTION
**Pull request by lauraturk**
### What
* Added SpeechRec component to recognize a word and update game input value w/ it. Works with the game mechanics already set up.
* Added SpeechSynth component to create a verbal prompt for the word-type required.
* Added play and listen to game input state to manage when the vocal prompt is on and off and start speech recognition.
* Added mic and speaker icons in an asset folder
* Added toggle to app state to initiate a verbal game or not. Can be toggled mid-game.

### Next
* Some life-cycle niceties should be added to ensure we are not re-rendering unnecessarily.
* convert speechsynth to a stateless component.

### CODE REVIEW

**Comment by lauraturk** [comment]
@anderswood @cbandrow 
🌮 🌮 🌮

File: public/index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>MadTalk</title>
```
Comment (cbandrow-zz) [inline public/index.html:22]: 🌟

File: public/manifest.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "MadTalk",
+  "name": "MadTalk",
```
Comment (cbandrow-zz) [inline public/manifest.json:3]: ⭐️ ⭐️

File: src/components/App.js [hunk 3]
```diff
@@ -62,10 +58,12 @@ class App extends Component {
       <section className="App">
         <section className="text-source-container">
           <h2>new route: (?) section to select text source</h2>
+          <button className="App-speechBtn" onClick={() => this.setState({speechEnabled: !this.state.speechEnabled})}>{!this.state.speechEnabled ? icons.micOff : icons.micOn }</button>
```
Comment (cbandrow-zz) [inline src/components/App.js:61]: So this button is present on the screen the whole time and enables speech? Works for me, I'll figure out a way to enable it 100% of the time for conditional rendering too.

**Comment by cbandrow-zz** [comment]
Looks awesome Laura! Merging it to staging!

**Pull request merged**